portal-impl/src/com/liferay/portlet/asset/service/impl/AssetEntryLocalServiceImpl.java

```diff
@@ -450,12 +455,11 @@ public AssetEntry incrementViewCounter(
 		return assetEntry;
 	}
 
-	@BufferedIncrement(
-		configuration = "AssetEntry", incrementClass = NumberIncrement.class
-	)
 	@Override
+	@Transactional(enabled = false)
 	public void incrementViewCounter(
-		long userId, String className, long classPK, int increment) {
+		long companyId, long userId, String className, long classPK,
+		int increment) {
```

Comment:
@stsquared99 this "increment" parameter is not used at all.
This convenient method https://github.com/shuyangzhou/liferay-portal/pull/8416/files#diff-5607c7ad65ea9f759fa3f371e303a802R29 should be removed. When not having it, it won't be possible to make mistakes like this.

Please remove that method regenerate, cleanup exist usages and resend.